osu.Game/Rulesets/Difficulty/DifficultyAttributes.cs

```diff
@@ -30,6 +30,8 @@ public class DifficultyAttributes
         protected const int ATTRIB_ID_AIM_DIFFICULT_STRAIN_COUNT = 25;
         protected const int ATTRIB_ID_OK_HIT_WINDOW = 27;
         protected const int ATTRIB_ID_MONO_STAMINA_FACTOR = 29;
+        protected const int ATTRIB_ID_AIM_DIFFICULT_SLIDER_COUNT = 31;
```

Comment:
Are these new attributes documented somewhere? Which ruleset are they for? Should they be persisted to the database (apparently some shouldn't now?)

Reply:
This is one of the osu's new attributes, I'm not sure how we should document them code wise other than just through the usage